### ISSUE TITLE
Running out of training time

### ISSUE THREAD

**Issue by beccamc** [dotnet/machinelearning-modelbuilder#584]
I run out of training time on the [Titanic dataset](https://www.kaggle.com/c/titanic/data). A few thoughts here... 

1. This is a popular dataset (starter dataset for Kaggle) and we should look into why our time prediction is too low. It'd be nice to make this more accurate.

2. I think the "Training time finished" should be an expected exception (shows the error warning dialog without stack trace.)  This is expected and we should update the message to say something like "Training time finished without any model trained. Try training for longer." Including the stack trace makes it look like something went wrong.

3. Why do I have an accuracy if the training didn't finish? Do we have a trained model but we haven't finished some step so its failing the whole run?

![image.png](https://images.zenhubusercontent.com/5d55c3d68fac840001314eaa/e0074d31-b893-4f7f-ae61-3db46bd33d98)

Dataset:
[https://app.zenhub.com/files/186917455/5ea0b0dc-b431-436f-9ad9-55648ac93bb5/download](https://app.zenhub.com/files/186917455/5ea0b0dc-b431-436f-9ad9-55648ac93bb5/download)

**Comment by zewditu** [comment]
The same thing for me and there is also customer report  https://app.zenhub.com/workspaces/mlnet-tools-5cde1c97e3106e39e8ae08fc/issues/dotnet/machinelearning-modelbuilder/596

**Comment by LittleLittleCloud** [comment]
Accuracy after training is finished should be fixed after commit id 0d89134b1e1(Fix Dup Output), let me know if this bug appears after that commit

**Comment by beccamc** [comment]
1. Tracking getting better training times in #672. Titanic dataset is mentioned 
2.  I updated the error message so it now looks like this ![image.png](https://images.zenhubusercontent.com/5d55c3d68fac840001314eaa/26624f8d-438f-4d2c-a3af-2d2666c9f9ad)
3. I haven't seen this problem recently, looks like Xiaoyun fixed it. 

Closing directly as 2 is the only one that can be verified, and it already had been with another work item.
